src/librustc/diagnostics.rs

```diff
@@ -2011,53 +2011,6 @@ a (non-transparent) struct containing a single float, while `Grams` is a
 transparent wrapper around a float. This can make a difference for the ABI.
 "##,
 
-E0700: r##"
```

Comment:
I don't think we outright remove error code descriptions we don't emit anymore (at least not if they have hit stable, I'm sure).